gzlib.c

```diff
@@ -6,7 +6,7 @@
 #include "zbuild.h"
 #include "gzguts.h"
 
-#if defined(WIN32) && !defined(__BORLANDC__)
+#if defined(_WIN32) && !defined(__BORLANDC__)
```

Comment:
```_WIN32``` and ```__BORLANDC__``` are mutually exclusive... Borland C use different preprocessor macro for 32-bit Windows.

Reply:
it uses __WINS__ and if __WINS__ is defined then _WIN32 gets defines also.
But all of that can/will be removed anyways. So, checking for _WIN32 instead of WIN32 is still ok.

Reply:
> _WIN32 and __BORLANDC__ are mutually exclusive...

No, it's not!

_WIN32 is not _MSC_VER. Are you again confusing it?

Reply:
I'm not confusing, you're making a mess.